### PR TITLE
Read me update

### PR DESCRIPTION
**Pull request by InderKhera**
Changes to READ.me include:
- adding terminal commands
- adding dependencies 
- removing conda option to have READ.me in line with tiffany's example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ as well as The Diabetes Pedigree Function (which is a score that gives an idea a
 
 The final report can be found [here](https://github.com/UBC-MDS/diabetes_predictor_py/tree/main/analysis) or [this webpage](https://ubc-mds.github.io/diabetes_predictor_py/).
 
+## Dependencies
+- [Docker](https://www.docker.com/) 
+- [VS Code](https://code.visualstudio.com/download)
+- [VS Code Jupyter Extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
+
```
Comment (JennyonOort) [inline README.md:33]: I believe we still need `nb_conda_kernel`, `mamba`, `conda`, and `conda-lock`, please refer to our dependencies section from milestone 2 and add.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ as well as The Diabetes Pedigree Function (which is a score that gives an idea a
...
 ## Usage
 
 To replicate this analysis, follow the steps below. You can run the analysis using one of two methods: **Docker** or **Conda**.
```
Comment (JennyonOort) [inline README.md:36]: We are changing this section to Docker ONLY.

File: README.md [hunk 3]
```diff
@@ -55,54 +60,50 @@ cd diabetes_predictor_py
...
+    --validated-data=data/processed/df.csv \
+    --data-to=data/processed \
+    --plot-to=results/figures
 
-### Option 2: Using Conda
+    python scripts/split_dataset.py \
+     --input-file ./data/processed/train_df.csv \
+      --output-dir ./data/processed/
 
-1. Set up the Conda environment and run JupyterLab using the provided script:
```
Comment (JennyonOort) [inline README.md:72]: This Conda part can be deleted.

File: README.md [hunk 4]
```diff
@@ -55,54 +60,50 @@ cd diabetes_predictor_py
...
-These steps ensure you can run the analysis seamlessly using either Docker or Conda.
+---
 
 ---
 ### Clean up
 
 1. Docker: Type `Ctrl` + `C` in the terminal where you launched the container, 
 and then type `docker compose rm` to shut down the container and clean up the resources
 
-2. Conda: Type `Ctrl` + `C` in the terminal where Jupyter Notebook is launched,
```
Comment (JennyonOort) [inline README.md:101]: Delete tas part of conda.

**Review by JennyonOort — CHANGES_REQUESTED** [review]
Please see comments below. Mostly to reflect what the team just decided on. 

Note, I would double check the numbers below in Quarto report. For example, I believe should be 53 incorrections, than 46. 

```
Our final model performed decent on an unseen test dataset, achieving an overall accuracy of 0.75. Out of 218 test cases, the model correctly identified 164. 
However, it made 46 incorrect predictions, of which, 19 are false positives - incorrectly classifying non-diabetic subjects to diabetic- 
and 35 are false negatives - fail to diagnose diabetes when the patient is actually diabetic.
```

File: README.md (hunk 1 above)
Comment (InderKhera) [reply README.md:33]: These dependencies exist under  
## Developer Dependencies (starting line 106)

- conda (version 23.9.0 or higher)
- conda-lock (version 2.5.7 or higher)
- mamba (version 1.5.8 or higher)
- jupyterlab (version 4.0.0 or higher)
- nb_conda_kernels (version 2.3.1 or higher)
- Python and packages listed in [`environment.yml`](https://github.com/UBC-MDS/diabetes_predictor_py/blob/main/environment.yml)

File: README.md [hunk 5]
```diff
@@ -7,9 +7,9 @@ Inder Khera, Jenny Zhang, Jessica Kuo, Javier Martinez (alphabetically ordered)
 ## About
 
 In this study, we aim to develop a classification model using the logistic regression (LR) algorithm to predict whether a patient is expected to have diabetes or not. 
-Our final model performed decent on an unseen test dataset, achieving an overall accuracy of 0.75. Out of 218 test cases, the model correctly identified 164. 
-However, it made 46 incorrect predictions, of which, 19 are false positives - incorrectly classifying non-diabetic subjects to diabetic- 
-and 35 are false negatives - fail to diagnose diabetes when the patient is actually diabetic. 
+Our final model performed decent on an unseen test dataset, achieving an overall accuracy of 0.80. Out of 101 test cases, the model correctly identified 81.
```
Comment (JennyonOort) [inline README.md:10]: I think we can include both the results from train and test. In fact, can we do something similar to the below from Tiff's example? I think all the write ups can be grabbed somewhere in Quarto, just need to concatenate.

```
Here we attempt to build a classification model using the k-nearest neighbours algorithm which can use breast cancer tumour image measurements to predict whether a newly discovered breast cancer tumour is benign (i.e., is not harmful and does not require treatment) or malignant (i.e., is harmful and requires treatment intervention). Our final classifier performed well on an unseen test data set, with the F2 score, where beta = 2, of 0.97 and an overall accuracy calculated to be 0.87. On the 171 test data cases, it correctly predicted 157. Nine mistakes were predicting a benign tumour as malignant, while 4 mistakes where predicting a malignant tumour as benign. This is somewhat promising for implementing this in the clinic as false positives are less harmful than false negatives. Although they could theoretically cause the patient to undergo unnecessary treatment if the model is used as a decision tool, it is likely that the model is used for initial screening and that there will be a follow up appointment and further testing until treatment commences. However, the observation of even 4 mistakes predicting a malignant tumour as benign is concerning. As such, we believe further development of this model is needed for it to have clinical utility. Research to improve the model performance and understand the characteristics of incorrectly predicted patients is recommended.
```

**Review by JennyonOort — CHANGES_REQUESTED** [review]
Just one thought on About part. Otherwise good to go.

**Pull request merged**